<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneHierarchyWidget</class>
 <widget class="QWidget" name="SceneHierarchyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#sceneTree {
	background-color: #202020;
	border-top: 1px solid #111;
}

#addBtn, #deleteBtn {
	padding: 4px;
}

#addBtn {
	border: 0;
}

#deleteBtn {
	border-left: 1px solid #333;
}

QTreeWidget {
  outline: none;
  selection-background-color: #404040;
  color: #CECECE;
}

QTreeWidget::item {
	padding: 6px;
}

QTreeWidget::item:selected {
	selection-background-color: #404040;
	background: #404040;
	outline: none;
  }


/* important when the widget loses focus */
QTreeWidget::item:selected:!active {
	background: #404040;
	padding: 0;
	color: #CECECE;
}

QTreeWidget::item:selected:active {
	background: #404040;
	padding: 0;
}

QTreeWidget::item:hover {

}
  </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	border: none;
	background-color: #303030;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="addBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset resource="../../app/icons.qrc">
          <normaloff>:/icons/add_object.svg</normaloff>:/icons/add_object.svg</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="../../app/icons.qrc">
          <normaloff>:/icons/delete_object.svg</normaloff>:/icons/delete_object.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="sceneTree">
     <property name="styleSheet">
      <string notr="true">outline: none</string>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>0</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../app/icons.qrc"/>
 </resources>
 <connections/>
</ui>
